web/app/i18n/fr.json

```diff
@@ -0,0 +1,321 @@
+{
+    "LANG": "Français",
+    "AGG": {
+        "DAY": "Jour",
+        "DAY_OF_MONTH": "Jour du Mois",
+        "DAY_OF_WEEK": "Jour de la semaine",
+        "FILTER": "Filtre",
+        "GEOGRAPHY": "Géographie",
+        "HOUR_OF_DAY": "Heure du jour",
+        "MONTH": "Mois",
+        "MONTH_OF_YEAR": "Mois de l'année",
+        "TIME": "Temps",
+        "WEEK": "Semaine",
+        "WEEK_OF_YEAR": "Semaine de l'année",
+        "YEAR": "Année"
+    },
+    "COMMON": {
+        "ADD": "Ajouter",
+        "APP_BRAND": "Système de données sur les accidents de la route",
+        "APP_NAME": "DRIVER",
+        "CANCEL": "Annuler",
+        "CLOSE": "Fermer",
+        "DATE_RANGE": "Plage de dates",
+        "CREATED_RANGE": "!Created Date",
```

Comment:
Any reason for leaving this one out?

Reply:
It wasn't provided in the translation but seems straightforward enough to put in. It was missing because this was added after I created the translation spreadsheet for the WB. I've added the missing ones to the sheet and will request updates from the WB.